modules/ibi-extra-manifests-configmap.adoc

```diff
@@ -121,9 +121,12 @@ cA8SMRwpUbz3LXY=
 +
```

Comment:
In the preview I noticed that the instruction[1] is missing back ticks for ConfigMap. While you are editing this module can you address that too?

[1] ". Create the `ConfigMap` object by running the following command:"